File: python/pytorch/train/vgg16/train_cifar100.py
from python.pytorch.models.vgg16 import run_experiment

if __name__ == "__main__":
    run_experiment(
        dataset_path="data/cifar100_png",
        output_file="vgg16_cifar100.csv",
        checkpoint_path="checkpoints/vgg16_cifar100.pth",
        img_size=(32, 32),
        grayscale=False
    )

File: R/train/vgg/train_cifar100.r
source("R/models/vgg16.r")
cat("Script avviato!\n")
run_experiment(
  dataset_path = "data/cifar100_png",
  checkpoint_path = file.path("R/checkpoints/vgg16_cifar100_r.pt"),
  img_size = c(32, 32),
  grayscale = FALSE,
  test_split = "test",
  epochs = 30,
  batch_size = 128
)